app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -236,7 +235,7 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         }
     }
 
-    override fun onNonTextInputUrlEntered(urlStr: String) = with(navUrlInput) {
+    override fun onNonTextInputUrlEntered(urlStr: String) = with (navUrlInput) {
```

Comment:
nit: extra space - does the linter catch this?

Reply:
Why do you need to have this in a separate function, rather than directly in the loadUrl interface? Can you explain why you did it this way?

Reply:
^why not directly into loadUrl interface?